devicetypes/fibargroup/fibaro-door-window-sensor-zw5.src/fibaro-door-window-sensor-zw5.groovy

```diff
@@ -18,11 +18,12 @@ metadata {
 		capability "Battery"
 		capability "Contact Sensor"
 		capability "Sensor"
-        capability "Configuration"
-        capability "Tamper Alert"
+        	capability "Configuration"
+        	capability "Tamper Alert"
 		capability "Health Check"
 
-        fingerprint deviceId: "0x0701", inClusters: "0x5E, 0x85, 0x59, 0x22, 0x20, 0x80, 0x70, 0x56, 0x5A, 0x7A, 0x72, 0x8E, 0x71, 0x73, 0x98, 0x2B, 0x9C, 0x30, 0x86, 0x84", outClusters: ""
+		fingerprint mfr: "010F", prod: "0700"
+		fingerprint mfr: "010F", prod: "0701"
```

Comment:
I see three versions of this FGK-10X sensor on the z-wave alliance site:

https://products.z-wavealliance.org/products/1356?selectedFrequencyId=-1
https://products.z-wavealliance.org/products/1620?selectedFrequencyId=-1
https://products.z-wavealliance.org/products/1050?selectedFrequencyId=-1

so it looks like there are two EU versions and we're missing the `0702` one

Reply:
`0702` has its own specific DTH. 

https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/fibargroup/fibaro-door-window-sensor-2.src/fibaro-door-window-sensor-2.groovy

If both DTHs are the same, I would say go ahead and consolidate into one. 

@jaemin11lee, Can you look at the differences between both DTHs?

Reply:
It has different capability. handling with another dth looks reasonable for me.

Reply:
So, you are suggesting to keep the DTHs separate?

Reply:
OK, go ahead with the merge then.